app/src/main/java/de/danoeh/antennapod/fragment/ItemlistFragment.java

```diff
@@ -638,14 +646,15 @@ private void loadItems() {
                 });
     }
 
-    private Pair<Feed, LongList> loadData() {
+    private Object[] loadData() {
         Feed feed = DBReader.getFeed(feedID);
         if(feed != null && feed.getItemFilter() != null) {
             FeedItemFilter filter = feed.getItemFilter();
             feed.setItems(filter.filter(feed.getItems()));
         }
         LongList queuedItemsIds = DBReader.getQueueIDList();
-        return Pair.create(feed, queuedItemsIds);
+        LongList favoritedItemsId = DBReader.getFavoriteIDList();
```

Comment:
Ah, you're right.  I suppose we can take the time to load the list if they've done a long click.